server/BREAKING.md

```diff
@@ -0,0 +1,31 @@
+
+## 0.1020 Breaking Changes
+
+- [`RestWrapper` is now an abstract class](#`restwrapper`-is-now-an-abstract-class)
+- [`Historian` no longer handles request headers](#`historian`-no-longer-handles-request-headers)
+
+### `RestWrapper` is now an abstract class
+
+`RestWrapper` is now an abstract class that cannot be instantiated. Use `BasicRestWrapper` instead to maintain current functionality.
+
+### `Historian` no longer handles request headers
+
+The `Historian` client class no longer builds its own request headers, and therefore does not have constructor parameters `getCredentials` and `getCorrelationId`. Instead, it relies on the consumer to pass in a `RestWrapper` with the desired default headers. To easily generate the necessary token format for communicating with the Historian service, use the new `getAuthorizationTokenFromCredentials()` function. For example,
```

Comment:
Note that historian has a different release cadence and number then routerlicious.

The latest published one on NPM is 0.0.10995.....

Reply:
Is this a breaking change? We are only updating client code, even in the context when scribe uses it. The contract between client and service remains the same right? In other words, both older and newer historian image should work with this change right?

Reply:
Is this a breaking change? If I am creating a new instance of r11s driver, do I need to do anything differently?

Reply:
This `Historian` class is part of `@fluidframework/server-services-client` from the routerlicious folder. The confusion  (seen here) between server and package releases by trying to combine them in 1 server/breaking.md was my main reason for wanting a server/routerlicious/breaking.md

Reply:
> Is this a breaking change? If I am creating a new instance of r11s driver, do I need to do anything differently?

It's a breaking change to using the Historian (and RestWrapper class) when consuming `@fluidframework/server-services-client`. This will affect the internal implementation of r11s driver once we rely on this version there, but should not affect consumers of `@fluidframework/routerlicious-driver` so long as they use DocumentServiceFactory instead of rolling their own DocumentStorageService

Reply:
Makes sense. Once the client bumps, it will need to upgrade as well.